scheduler/test/e2e/fake/v1alpha2/keptnworkloadinstance_types.go

```diff
@@ -45,36 +45,27 @@ type KeptnWorkloadInstanceStatus struct {
 	// +kubebuilder:default:=Pending
 	PostDeploymentEvaluationStatus common.KeptnState `json:"postDeploymentEvaluationStatus,omitempty"`
 	// +kubebuilder:default:=Pending
-	PostDeploymentStatus               common.KeptnState  `json:"postDeploymentStatus,omitempty"`
-	PreDeploymentTaskStatus            []TaskStatus       `json:"preDeploymentTaskStatus,omitempty"`
-	PostDeploymentTaskStatus           []TaskStatus       `json:"postDeploymentTaskStatus,omitempty"`
-	PreDeploymentEvaluationTaskStatus  []EvaluationStatus `json:"preDeploymentEvaluationTaskStatus,omitempty"`
-	PostDeploymentEvaluationTaskStatus []EvaluationStatus `json:"postDeploymentEvaluationTaskStatus,omitempty"`
-	StartTime                          metav1.Time        `json:"startTime,omitempty"`
-	EndTime                            metav1.Time        `json:"endTime,omitempty"`
-	CurrentPhase                       string             `json:"currentPhase,omitempty"`
+	PostDeploymentStatus               common.KeptnState `json:"postDeploymentStatus,omitempty"`
+	PreDeploymentTaskStatus            []ItemStatus      `json:"preDeploymentTaskStatus,omitempty"`
+	PostDeploymentTaskStatus           []ItemStatus      `json:"postDeploymentTaskStatus,omitempty"`
+	PreDeploymentEvaluationTaskStatus  []ItemStatus      `json:"preDeploymentEvaluationTaskStatus,omitempty"`
+	PostDeploymentEvaluationTaskStatus []ItemStatus      `json:"postDeploymentEvaluationTaskStatus,omitempty"`
+	StartTime                          metav1.Time       `json:"startTime,omitempty"`
+	EndTime                            metav1.Time       `json:"endTime,omitempty"`
+	CurrentPhase                       string            `json:"currentPhase,omitempty"`
 	// +kubebuilder:default:=Pending
 	Status common.KeptnState `json:"status,omitempty"`
 }
 
-type TaskStatus struct {
-	TaskDefinitionName string `json:"taskDefinitionName,omitempty"`
+type ItemStatus struct {
+	DefinitionName string `json:"definitionName,omitempty"`
 	// +kubebuilder:default:=Pending
 	Status    common.KeptnState `json:"status,omitempty"`
-	TaskName  string            `json:"taskName,omitempty"`
+	Name      string            `json:"name,omitempty"`
```

Comment:
maybe it makes sense to add some documentation comments here to make the difference between `Name` and `DefinitionName` more clear